Fabric.Authorization.API/Modules/GroupsModule.cs

```diff
@@ -439,5 +440,20 @@ private Negotiator ValidateGroupUserRequest(GroupUserRequest groupUserRequest)
             }
             return null;
         }
+

```

Comment:
Can this method also be used in the `AddRolesToGroup` function? Perhaps it would need a minor refactor to accomplish that.

Reply:
We could potentially extract a function that iterates through roles and checks write access. I'll refactor that.